internal/config/main.go

```diff
@@ -2,17 +2,29 @@ package config
 
 import (
 	"fmt"
-
+	"github.com/calvinbui/homer-docker-service-discovery/internal/consul"
 	"github.com/calvinbui/homer-docker-service-discovery/internal/docker"
 	"github.com/calvinbui/homer-docker-service-discovery/internal/logger"
 	"github.com/docker/docker/client"
+	"github.com/hashicorp/consul/api"
 
 	"github.com/caarlos0/env/v6"
 )
 
+const (
+	Docker = "Docker"
+	Consul = "Consul"
+)
```

Comment:
I dont find this necessary, unless you have a reason?

Reply:
I just would like test to create something like an enumeration to avoid to modify all the if the configuration variable change